### PR TITLE
Readme updates for systemd  timers

### PR DESCRIPTION
**Pull request by MartyEwings**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,18 +38,26 @@ By default you get the following:
 ## Usage
 
 In order to use this module, classify the node running PE PostgreSQL with the `pe_databases` class.
-That node is the Primary Master in a Monolithic installation, or the PE PuppetDB host in a Split install.  
+That node is the Primary Server in most instances, but there may be one or more Servers with this role in an XL deployment
```
Comment (jarretlavallee) [inline README.md:41]: We should also note the replica as well. Does `Servers` need to be capitalized?

File: README.md [hunk 2]
```diff
@@ -40,18 +38,26 @@ By default you get the following:
 ## Usage
 
 In order to use this module, classify the node running PE PostgreSQL with the `pe_databases` class.
-That node is the Primary Master in a Monolithic installation, or the PE PuppetDB host in a Split install.  
+That node is the Primary Server in most instances, but there may be one or more Servers with this role in an XL deployment  
 
 To classify via the PE Console, create a new node group called "PE Database Maintenance".
```
Comment (jarretlavallee) [inline README.md:43]: This note doesn't even say to add the class.

File: README.md [hunk 3]
```diff
@@ -40,18 +38,26 @@ By default you get the following:
...
 ## Items you may want to configure
 
 ### Backup Schedule
 
+Backups are not activated by default but can be enabled by setting the following parameter:
+
+Hiera classification example 
+
+```
+pe_databases::manage_database_backups:: false
```
Comment (jarretlavallee) [inline README.md:56]:
```suggestion
pe_databases::manage_database_backups: false
```

File: README.md [hunk 4]
```diff
@@ -65,12 +71,10 @@ You can configure the retention policy by setting `pe_databases::backup::retenti
 
 ### Disable Maintenance
 
-The maintenance cron jobs will perform a `VACUUM FULL` on various `pe-puppetdb` tables to keep them lean and fast.
-A `VACUUM FULL` is a blocking operation and you will see the PuppetDB command queue grow while the cron jobs run.
-The blocking should be short lived and the PuppetDB command queue should work itself down after, however, if for some reason you experience issues you can disable the maintenance cron jobs.
+The maintenance SystemD timers will perform a `pg_repack` on various `pe-puppetdb` tables to keep them lean and fast.
+A `pg_repack` is a non-blocking maintence action, however, if for some reason you experience issues you can disable the maintenance SystemD timers,
 You can do so by setting `pe_databases::maintenance::disable_maintenance: true` in your hieradata.
```
Comment (jarretlavallee) [inline README.md:76]: This is a parameter for a class that doesn't exist. @m0dular We need to add this parameter to the main class and check for both.

**Review by jarretlavallee — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (MartyEwings) [reply README.md:43]: `In order to use this module, classify the node running PE PostgreSQL with the `pe_databases` class.` is that not sufficient? should we put in a hiera reference or something

Comment (jarretlavallee) [reply README.md:43]: Yeah. I just saw that this whole note says to create a node group and pin the node, but never says to add the class to the node group. However, I think we should just remove this section with the idea of auto inclusion.

**Comment by MartyEwings** [comment]
closing this lost the thread on the updates! starting again

**Pull request closed without merging**